### PR TITLE
Add Turing projects

### PR DESCRIPTION
**Pull request by cpfiffer**
Adds Turing projects for GSoD.

### CODE REVIEW

File: jsoc/gsod/projects.md [hunk 1]
```diff
@@ -169,3 +169,21 @@ __Potential impact and the Why?:__
 ## Queryverse
 
 [Queryverse](https://www.queryverse.org/) is a family of packages in the data science space. Many packages have rudimentary documentation but could use updates, polish, proper tutorials etc. We are looking for help with documentation for all packages, and to build out the homepage for the project. The mentor for this project would be [David Anthoff](https://github.com/davidanthoff).
+
+## Turing
+
+[Turing.jl](https://turing.ml/dev/) is a probabilistic programming language written in Julia. The team is looking for help with several projects. Mentors for this project would be [Cameron Pfiffer](https://github.com/cpfiffer/), [Martin Trapp](https://github.com/trappmartin/), or [Hong Ge](https://github.com/yebai/].
```
Comment (cpfiffer) [inline jsoc/gsod/projects.md:175]:
```suggestion
[Turing.jl](https://turing.ml/dev/) is a probabilistic programming language written in Julia. The team is looking for help with several projects. Mentors for this project would be [Cameron Pfiffer](https://github.com/cpfiffer/), [Martin Trapp](https://github.com/trappmartin/), or [Hong Ge](https://github.com/yebai/).
```

File: jsoc/gsod/projects.md [hunk 2]
```diff
@@ -169,3 +169,21 @@ __Potential impact and the Why?:__
...
+
+- A better tutorial that shows how to integrate [Flux.jl](https://github.com/FluxML/Flux.jl) with Turing. Turing has an [existing tutorial](https://turing.ml/dev/tutorials/3-bayesnn/) that shows how to build a Bayesian neural network with Flux.jl, but this is very much a toy example and does not demonstrate real-world applicability. A guide that shows how to apply Flux and Turing in a useful setting would be valuable for Julia, Turing, and Flux.
+
+- Documentation and manuals for [MCMCChains](https://github.com/TuringLang/MCMCChains.jl), [AdvancedHMC](https://github.com/TuringLang/AdvancedHMC.jl), and [Bijectors](https://github.com/TuringLang/Bijectors.jl). Turing maintains several separate packages, all of which have some documentation, but each of these could be improved dramatically.
```
Comment (logankilpatrick) [inline jsoc/gsod/projects.md:185]: Can you detail how they could be improved? What would like steps be to do this? something like: 
 - evaluate current docs 
- meet with core team to understand the high-level goals of the package
- verify the docs meet those goals
- etc etc

**Comment by logankilpatrick** [comment]
This looks awesome. Only thing I would update would be more concreteness around the Documentation and manuals section

File: jsoc/gsod/projects.md (hunk 2 above)
Comment (cpfiffer) [inline jsoc/gsod/projects.md:185]:
```suggestion
- Documentation and manuals for [MCMCChains](https://github.com/TuringLang/MCMCChains.jl), [AdvancedHMC](https://github.com/TuringLang/AdvancedHMC.jl), and [Bijectors](https://github.com/TuringLang/Bijectors.jl). Turing maintains several separate packages, all of which have some documentation, but each of these could be improved dramatically. Technical writers would review current documentation, meet with the development team for each package to determine the package goals, and assess whether current documentation meets those goals. AdvancedHMC, for example, requires a much more detailed guide that explores all of AdvancedHMC's functionality.
```

@logankilpatrick how's this?

File: jsoc/gsod/projects.md [hunk 3]
```diff
@@ -169,3 +169,21 @@ __Potential impact and the Why?:__
 ## Queryverse
 
 [Queryverse](https://www.queryverse.org/) is a family of packages in the data science space. Many packages have rudimentary documentation but could use updates, polish, proper tutorials etc. We are looking for help with documentation for all packages, and to build out the homepage for the project. The mentor for this project would be [David Anthoff](https://github.com/davidanthoff).
+
+## Turing
+
+[Turing.jl](https://turing.ml/dev/) is a probabilistic programming language written in Julia. The team is looking for help with several projects. Mentors for this project would be [Cameron Pfiffer](https://github.com/cpfiffer/), [Martin Trapp](https://github.com/trappmartin/), or [Hong Ge](https://github.com/yebai/).
```
Comment (cpfiffer) [inline jsoc/gsod/projects.md:175]:
```suggestion
[Turing.jl](https://turing.ml/dev/) is a probabilistic programming language written in Julia. The team is looking for help with several projects. Mentors for this project would be [Cameron Pfiffer](https://github.com/cpfiffer/), [Martin Trapp](https://github.com/trappmartin/), [Kai Xu](https://github.com/xukai92), or [Hong Ge](https://github.com/yebai/).
```

File: jsoc/gsod/projects.md (hunk 2 above)
Comment (logankilpatrick) [reply jsoc/gsod/projects.md:185]: Looks good!

**Comment by cpfiffer** [comment]
Yeah, I'm happy with it. Thanks!

**Pull request merged**